scripts/sparse-and-shallow-git-fetch.sh

```diff
@@ -12,8 +12,6 @@ fi
 
 set -ev
 
-GIT="/usr/bin/git"
```

Comment:
I believe there were security reasons to use a hard-coded path for this dependency. @arobins do you recall that conversation?